src/app/ddQVTKWidgetView.cpp

```diff
@@ -110,7 +120,11 @@ ddQVTKWidgetView::ddQVTKWidgetView(QWidget* parent) : ddViewBase(parent)
   this->Internal->VTKWidget->SetRenderWindow(this->Internal->RenderWindow);
 #endif
   this->Internal->VTKWidget->SetRenderWindow(this->Internal->RenderWindow);
```

Comment:
@sankhesh do you agree we can use only the second call the SetRenderWindow?  this comes from this commit: https://github.com/RobotLocomotion/director/commit/dd4ee5d12ebbf505279d4248f2530ce5fc87c3b4#diff-ab3b6e434c7408392ed6672f3ef053d8

Reply:
@peteflorence Thanks for spotting this. I think that is just oversight. It would be fine to remove the `SetRenderWindow` call within the `#else` block.

Reply:
:+1: thanks